components/gantry/oneaxis/oneaxis.go

```diff
@@ -160,13 +164,65 @@ func newOneAxis(ctx context.Context, deps resource.Dependencies, conf resource.C
 		return nil, errors.Errorf("invalid gantry type: need 1, 2 or 0 pins per axis, have %v pins", np)
 	}
 
-	if err := oAx.home(ctx, np); err != nil {
+	if err := oAx.home(context.Background(), np); err != nil {
```

Comment:
@randhid is this all that's meant by [RSDK-3533](https://viam.atlassian.net/browse/RSDK-3533)? if so I thought it would be easy to just group in with this PR since it's dealing with different contexts already

Reply:
yep, thank you!